scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -127,6 +127,7 @@
     "\n",
```

Comment:
<p>Not sure we want to expose EXP_NAME here. How about we just set this as default in the summary function?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Will not change evaluate().